### DISCUSSION TITLE
nee more Excamples

### DISCUSSION THREAD

**Discussion by Axyxo** [Appsilon/rhino#513] [General]
hi everyone,
i need more rhino examples. i want to learn rhino but there are too less examples and resources available. at the moment i try to make a rhino app with Fluent. all i have is the Olympic medals example. i can say that it is not so easy for me to switch my progamming style from the classical shiny programming style to the modularization rhino style. i am sure that other people need also more resources for learning!!!
greeting

**Comment by jakubnowicki** [comment]
Hi @Axyxo ,
If you want to build a Rhino app with `shiny.fluent`, the best place to start will be [this tutorial](https://appsilon.github.io/shiny.fluent/articles/st-shiny-fluent-and-rhino.html).
